code/modules/augment/active/cyberbrain.dm

```diff
@@ -0,0 +1,117 @@
+/obj/item/organ/internal/augment/active/cyberbrain
+	name = "cyberbrain module"
+	action_button_name = "Access cyberbrain"
+	icon_state = "cyberbrain"
+	allowed_organs = list(BP_AUGMENT_HEAD)
+	augment_flags = AUGMENTATION_MECHANIC
+	var/list/stock_parts = list()
+	var/list/starting_stock_parts = list(
+		/obj/item/stock_parts/computer/processor_unit/small,
+		/obj/item/stock_parts/computer/hard_drive/silicon,
```

Comment:
Should be /obj/item/stock_parts/computer/hard_drive/small, silicon subtype comes preloaded with some programs people might not have access too.